src/stories/AnchoredOverlay.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import React, {useEffect, useRef, useState} from 'react'
+import {Meta} from '@storybook/react'
+
+import {BaseStyles, Box, ThemeProvider} from '..'
+import Heading from '../Heading'
+import DropdownButton from '../Button'
+import {AnchoredOverlay} from '../AnchoredOverlay'
+import {registerPortalRoot} from '../Portal'
+
+export default {
+  title: 'Generic behaviors/AnchoredOverlay',
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ]
+} as Meta
+
+const HeaderAndLayout = ({children, includePortal}: {children: JSX.Element; includePortal: boolean}) => {
+  const scrollingElementRef = useRef<HTMLDivElement>(null)
+  useEffect(() => {
+    if (scrollingElementRef.current && includePortal) {
+      registerPortalRoot(scrollingElementRef.current)
+    }
```

Comment:
This story doesn't appear to use `portalContainerName`.  Instead, it looks like it just registers the default portal within the scrolling element